src/client/components/Work/Detail/WorkDetail.component.js

```diff
@@ -157,13 +157,16 @@ export class WorkDetail extends React.Component {
     const lTitleFull = titleFull.toLowerCase();
 
     // Start with dcTitleFull
-    let res = lTitleFull;
+    let lRes = lTitleFull;
+    let res = titleFull;
 
     // Remove the multivolume title
-    res = res.replace(lTitle, '');
+    res = res.slice(lRes.indexOf(lTitle) + lTitle.length);
+    lRes = lRes.slice(lRes.indexOf(lTitle) + lTitle.length);
 
     // Remove volume indicator
-    res = res.replace(bind, '');
+    res = res.slice(lRes.indexOf(bind) + bind.length);
+    lRes = lRes.slice(lRes.indexOf(bind) + bind.length);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "lRes" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS1854)